src/api/devices.rs

```diff
@@ -318,3 +321,57 @@ impl Command<AuthenticateCommand> {
         Ok(())
     }
 }
+
+#[derive(Parser, Debug)]
+pub struct GetUpdateCommand {
+    /// The PRN of the device you wish to create the resource within.
```

Comment:
This comment also seems off - we're not creating anything. Maybe `The PRN of the device to check for an update`